.github/workflows/jdk8-postgres15-scan.yml

```diff
@@ -13,9 +13,21 @@ jobs:
     steps:
     - name: Checkout
       uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Login to GitHub Container Registry
+      uses: docker/login-action@v2
+      with:
+        registry: ghcr.io
+        username: ${{ github.repository_owner }}
+        password: ${{ secrets.CR_PAT }}
 
     - name: Build the Docker image locally for scanning
-      run: docker build ./jdk-postgres/8-15 -t can_be_removed
+      run: |
+        LAST_JDK8_VERSION=$(git tag | grep jdk8 | sed 's/jdk8-//g' | sort -r | head -n 1)
```

Comment:
```suggestion
        LAST_JDK8_VERSION=$(git tag | grep jdk8 | sed 's/jdk8-//g' | sort -r --version-sort | head -n 1)
```

I think we should add the `--version-sort` option to the `sort` command. Otherwise, there is a possibility that we get the wrong latest version.

You can see the details of the `--version-sort` option on the man page.
* https://man7.org/linux/man-pages/man1/sort.1.html
  * Excerpt
    ```
           -V, --version-sort
                  natural sort of (version) numbers within text
    ```

For example (I use `jre8` for testing), it returns `1.1.9` as the latest version even though the correct latest version is `1.1.14` as follows if we don't specify the `--version-sort`.

```shell
$ git tag | grep jre8 | sed 's/jre8-//g' | sort -r
1.1.9     # <- We will choose this version as the latest version by "head -n 1".
1.1.8
1.1.7
1.1.6
1.1.5
1.1.4
1.1.3
1.1.2
1.1.14
1.1.13
1.1.12
1.1.11
1.1.10
1.1.1
1.1.0
1.0.0
```

However, if we specify the `--version-sort`, we can get the correct latest version as follows.

```shell
$ git tag | grep jre8 | sed 's/jre8-//g' | sort -r --version-sort
1.1.14    # <- We will choose this version as the latest version by "head -n 1".
1.1.13
1.1.12
1.1.11
1.1.10
1.1.9
1.1.8
1.1.7
1.1.6
1.1.5
1.1.4
1.1.3
1.1.2
1.1.1
1.1.0
1.0.0
```

Reply:
@kota2and3kan 

Thank you so much. Sorry, I didn't know this option!
(I was thinking of using the `-n` option but forgot it...

I revised it in https://github.com/scalar-labs/docker/pull/23/commits/2e98569e395423c457a5e85445dd76e29dc4a6f7

Please take a look 🙇

Reply:
Thank you for the update!